src/edit/input.js

```diff
@@ -244,11 +248,67 @@ handlers.dragstart = (pm, e) => {
 
 handlers.dragend = pm => window.setTimeout(() => pm.input.dragginFrom = false, 50)
 
-handlers.dragover = handlers.dragenter = (_, e) => e.preventDefault()
+/**
+ * A cursor to indicate where text being dragged will be dropped into.
+ *
+ * Not sure where this class should go yet, here for context for now.
+ */
+class PseudoCursor {
+  constructor(pm) {
+    this.pm = pm
+    this.dom = pm.wrapper.appendChild(elt("div", {class: 'pseudo-cursor'}))
+  }
+
+  /**
+   * Update the visual location of the cursor.
+   *
+   * @param {Object} pos
+   */
+  updateLocation(pos) {
+    this.dom.style.left = pos.left + "px"
+    this.dom.style.top = pos.top + "px"
+    this.dom.style.height = pos.bottom - pos.top + "px"
+  }
+
+  remove() {
+    this.dom.parentNode.removeChild(this.dom)
+  }
+}
+
+insertCSS(`
+
+.pseudo-cursor {
+  position: absolute;
+  width: 2px;
+  background: #666;
+}
+
+`)
+
+// This should be debounced.
+handlers.dragover = handlers.dragenter = (pm, e) => {
+  e.preventDefault()
+  let cursorPos = pm.posAtCoords({left: e.clientX, top: e.clientY})
+  let coords = coordsAtPos(pm, cursorPos)
+  if ( ! pm.draggingCursor ) {
+    pm.draggingCursor = new PseudoCursor(pm)
+  }
+  let rect = pm.wrapper.getBoundingClientRect()
+  coords.top -= rect.top
+  coords.right -= rect.left
+  coords.bottom -= rect.top
+  coords.left -= rect.left
+  pm.draggingCursor.updatePos(coords)
+}
 
 handlers.drop = (pm, e) => {
   if (!e.dataTransfer) return
 
+  if (pm.draggingCursor) {
+    pm.draggingCursor.remove()
+    delete pm.draggingCursor
```

Comment:
Prefer setting a property to `null` rather than deleting it, to prevent changing the shape of the object again and again at runtime (which has a performance cost). Maybe even include this slot in the `input` object, from the start, so that we don't have to stick it into the main `pm` object.